config/locales/server.en.yml

```diff
@@ -141,10 +142,20 @@ en:
         username_taken: "That username is already taken on this server."
     auth:
       error:
-        invalid_oauth_domain: "Invalid ActivityPub OAuth domain"
-        oauth_session_expired: "ActivityPub OAuth session expired"
-        failed_to_authorize: "ActivityPub OAuth failed to authorize"
-        failed_to_get_actor: "ActivityPub OAuth failed to get actor"
+        not_enabled: "Auth is not enabled"
+        invalid_domain: "Invalid ActivityPub domain"
+        invalid_auth_redirect: "Invalid auth redirect"
+        invalid_redirect_params: "Invalid redirect params"
+        authorization_required: "Authorization record is required"
+        session_expired: "Auth session expired"
+        failed_to_authorize: "Failed to authorize"
+        failed_to_get_actor: "Failed to get actor"
+        failed_to_verify_redirect: "Redirect has not been added to allowed user api auth redirects"
```

Comment:
On the first point, I've added the auth_redirect and domain to the message: https://github.com/angusmcleod/discourse-activity-pub/commit/4c1d330015d6c212198868eae0a12749580fc8d4
It will now look like this:
```
https://test.localhost/ap/auth/redirect/discourse has not been added to allowed user api auth redirects on external1.com
```

Reply:
> Also, I was testing between an instance with this PR and an instance without this PR, and the Discourse-to-Discourse authorization process fails (which is expected) without an error.

I can't recreate this issue. When I attempt to authorize from an instance on this PR to one without the PR I get the appropriate message, regardless of whether the redirect is added to allowed user api auth redirects (as expected).

<img width="854" alt="Screenshot 2024-06-12 at 09 48 10" src="https://github.com/discourse/discourse-activity-pub/assets/5931623/0176635d-013d-4739-af12-6214b06004fc">

Reply:
Cool, thanks. This message is nicer! 

I am having a followup issue when testing this. Setup is the following: 

- site A has AP enabled for a while and is following a few other sources
- site B I just installed the AP plugin, enabled it and updated the `user api auth redirects`

When I try to authorize a user, the Verify step works fine, Authorize also goes to the right place in site B, but once I get redirected back to site A, I see this: 

<img width="815" alt="image" src="https://github.com/discourse/discourse-activity-pub/assets/368961/d24319a7-3877-474c-bef0-855e37502164">

is this because the user in site B does not have an AP actor yet? The plugin just got enabled there and no categories have Ap enabled, user hasn't had any posts published via AP.